dynamicweb/settings/prod.py

```diff
@@ -1,12 +1,8 @@
 from .base import *
 
-ADMINS = (
-    ('Nico Schottelius', 'nico.schottelius@ungleich.ch'),
-    ('Raul Ascencio', 'raul.ascencio@yandex.com'),
-    ('Web team', 'web-team@ungleich.ch')
+# List of people that get admin messages
+ADMINS = ( (x, x + "@ungleich.ch") for x in ["web-team"] )
```

Comment:
Might be a good idea to allow to define *additional* emails - I can fix that in a new PR later, I wanted to get rid of the plain mails first